src/Animations.ts

```diff
@@ -85,7 +86,7 @@ export const useTransition = (
       "useCode() is only available in Reanimated 1.0.0 or higher"
     );
   }
-  const { transitionVal, clock } = React.useMemo(
+  const { transitionVal, clock } = useMemoOne(
```

Comment:
What actually it does? It does not call more than once? If yes you can achieve it without package ;)

```js
const arePropsEqual = () => true

React.useMemo(..., arePropsEqual)
```

Reply:
it makes sure that the identity is preserved. React does not guarantee identity with `useMemo`. In the doc it says:

https://reactjs.org/docs/hooks-reference.html#usememo

```
You may rely on useMemo as a performance optimization, not as a semantic guarantee. In the future, React may choose to “forget” some previously memoized values and recalculate them on next render, e.g. to free memory for offscreen components. Write your code so that it still works without useMemo — and then add it to optimize performance.
```

Reply:
Ok! now that make sense :) And we can't do it by useRefs right?

Reply:
not really you would have to do a lazyUseRef kinda: https://twitter.com/dan_abramov/status/1149120970856075264
but at the end of the day useMemoOne does exactly what it need to do (as far as I reviewed the code)

Reply:
Ok :) so go for it!